bundles/pubsub/pubsub_admin_tcp/src/pubsub_tcp_handler.c

```diff
@@ -1116,8 +1117,7 @@ char *pubsub_tcpHandler_get_interface_url(pubsub_tcpHandler_t *handle) {
 //
 static inline
 int pubsub_tcpHandler_acceptHandler(pubsub_tcpHandler_t *handle, psa_tcp_connection_entry_t *pendingConnectionEntry) {
-    celixThreadRwlock_writeLock(&handle->dbLock);
-    // new connection available
+    // new connection available, requires dbLock to be write locked
```

Comment:
But there is no lock here?

Reply:
The function calling pubsub_tcpHandler_acceptHandler is responsible for already having locked it.

Reply:
OK, the comment confused me.